### ISSUE TITLE
Typography is not using the default Roboto font

### ISSUE THREAD

**Issue by IronGeek** [swordev/suid#165]
Hi, 

I'm not sure if this is intentional, but since **@suid/material v0.4.0** the `<Typography />` component is no longer using the default `font-family: "Roboto", "Helvetica", "Arial", sans-serif;`

For example, here's what the following snippets produces:

```jsx
import { Typography, Button } from '@suid/material';

export default function App() {
  return (<>
    <Typography>
      Should be using Roboto Font
    </Typography>
    <hr />
    <Button variant="contained">
      <Typography>
        Should be using Roboto Font
      </Typography>
    </Button>
    <hr />
    <Button variant="contained">Should be using Roboto Font</Button>
  </>);
}
```

- Using latest @suid/material v0.8.2 ([stackblitz.com/edit/angular-dgmcdj](https://stackblitz.com/edit/angular-dgmcdj?file=src%2FApp.tsx,package.json))

  ![image](https://user-images.githubusercontent.com/1383932/209463751-25a6ff83-88b8-4ca5-bd37-4fafaff02f45.png)

- Using @suid/material v0.3.1 ([stackblitz.com/edit/angular-32ictp](https://stackblitz.com/edit/angular-32ictp?file=src%2FApp.tsx,package.json))

  ![image](https://user-images.githubusercontent.com/1383932/209463770-5b1da1ea-6ad8-484d-b2fe-979d812ef044.png)

- For references, here's the Material UI version ([codesandbox.io/s/material-ui-forked-rteukt](https://codesandbox.io/s/material-ui-forked-rteukt?file=/index.tsx))

  ![image](https://user-images.githubusercontent.com/1383932/209463778-7ef7b3cf-18d7-4298-a788-510bb3c400b2.png)

---

**Update**

it seems all default styling (not only `font-family`) is not applied to `<Typography />` component.

The following are taken directly from the example at [suid.io/components/typography](https://suid.io/components/typography): [stackblitz.com/edit/angular-qgyehp](https://stackblitz.com/edit/angular-qgyehp?file=src%2FApp.tsx)

![image](https://user-images.githubusercontent.com/1383932/209477518-f9cd0aa4-612e-49f3-96ce-a79b29ee8d77.png)

**Comment by juanrgm** [comment]
It's a SolidJS 1.6.6 bug (https://github.com/solidjs/solid/issues/1446).

**Comment by juanrgm** [comment]
Closed via c14f863995a749b13d76e3e3e459e2b6f4b069f2.
